test/integration/backend-basic/fixture/toolpad/pages/page1/page.yml

```
@@ -93,3 +120,13 @@ spec:
       query:
         function: withParams
         kind: local
+    - name: manualQuery
+      mode: mutation
+      query:
+        function: manualQueryWithParams
+        kind: local
+      parameters:
+        - name: foo
+          value:
+            $$jsExpression: |
+              text.value.split(" ")?.[1]
```

Let's keep the binding simple? Any js expression will do.

```suggestion
              'checksum'
```